app/views/case_contacts/_case_contact.html.erb

```diff
@@ -31,12 +31,16 @@
   <td>
     <span class="mobile-label">Creator</span>
     <% if policy(contact).edit? %>
-      <% if contact.creator&.supervisor? %>
-        <%= link_to "#{contact.creator&.display_name} ", edit_supervisor_path(current_user) %>
-      <% elsif contact.creator&.casa_admin? %>
-        <%= link_to "#{contact.creator&.display_name} ", edit_users_path %>
+      <% if current_user.volunteer? %>
+        <%= contact.creator&.display_name %>
       <% else %>
-        <%= link_to "#{contact.creator&.display_name} ", edit_volunteer_path(contact.creator) %>
+        <% if contact.creator&.supervisor? %>
+          <%= link_to "#{contact.creator&.display_name} ", edit_supervisor_path(current_user) %>
+        <% elsif contact.creator&.casa_admin? %>
+          <%= link_to "#{contact.creator&.display_name} ", edit_users_path %>
+        <% else %>
+          <%= link_to "#{contact.creator&.display_name} ", edit_volunteer_path(contact.creator) %>
+        <% end %>
```

Comment:
@metamoni for all of these cases, I know the code was like this before, but can you remove the quotes with the extra space? It's not necessary:


```
<%= link_to contact.creator.decorate.name, edit_volunteer_path(contact.creator) %>
```

etc.

Reply:
I've been dying to fix those, just didn't want to seem like I'm fiddling with things out of scope haha. Great, will do!